properties.schema

```diff
@@ -137,6 +137,15 @@
                   "help": "Controls whether the assessment should be reset automatically (up to the number of available attempts) when a user revisits the page.",
                   "validators": []
                 },
+                "_allowResetIfPassed": {
+                  "type": "boolean",
+                  "required": true,
+                  "default": false,
+                  "title": "allow to retrying on Passed Assessment",
+                  "inputType": "Checkbox",
+                  "help": "Controls whether the assessment allow to be reset even once passed and even if there are attempts remaining.",
```

Comment:
The English here isn't great. ``title`` and ``help`` don't read well.
```json
"title": "Allow retry after assessment is passed.",
"help": "Controls whether the assessment is allowed to be reset once passed and with attempts remaining."
```
^ Something like that should do it. Although I'm not the best person to correct this.

Reply:
no issues @oliverfoster i will update it.